metricflow/test/model/validations/test_data_warehouse_tasks.py

```diff
@@ -152,26 +126,20 @@ def test_validate_dimensions(  # noqa: D
     issues = dw_validator.validate_dimensions(model)
     # One isssue is created for the short circuit query failure, and another is
     # created for the sub task checking the specific dimension
-    print("\nModelValidationResults", issues)
```

Comment:
LOL.

Reply:
Nothing to see here 🙃 totally didn't accidentally leave that in sometime on a previous PR